src/app/frontend/replicasetlist/replicasetlist.scss

```diff
@@ -16,8 +16,8 @@
 
 .md-fab {
   &.kd-replicaset-deploy {
-    position: absolute;
+    position: fixed;
     right: 0;
-    top: -56px / 2 - $baseline-grid;
+    top: $toolbar-height-size-base - $baseline-grid * 5;
```

Comment:
Should this be `$toolbar-height-size-base - /** half of height */ $baseline-grid * 3.5 - /* margin */ $baseline-grid`?
That looks centrally aligned.